front/app/containers/Authentication/useSteps/index.ts

```diff
@@ -248,10 +248,22 @@ export default function useSteps() {
         window.history.replaceState(null, '', '/');
       }
 
-      transition(currentStep, 'RESUME_FLOW_AFTER_SSO')();
+      const enterClaveUnicaEmail =
+        !isNilOrError(authUser) && isNil(authUser.attributes.email);
```

Comment:
I wonder if there's a better way to check this in case we have other SSO methods without emails in the future :thinking: A question for later though I think.

Reply:
Definitely. For now, we wanted to keep it clear that it's related exactly and only to ClaveUnica.